README.md

```diff
@@ -1621,6 +1621,11 @@ The following extractors use this feature:
 #### gamejolt
 * `comment_sort`: `hot` (default), `you` (cookies needed), `top`, `new` - choose comment sorting mode (on GameJolt's side)
 
+#### hotstar
+* `res`: resolution to ignore - one or more of `sd`, `hd`, `fhd`
```

Comment:
shouldn't it be `ignore-res` etc?

Reply:
Tooooo long to type. More importantly... extractor args is for power users anyways ( explains why they are documented in the second half of the readme) so naming shouldn't be a concern as it's documented properly.

Reply:
_suggestion: `igres`_